### DISCUSSION TITLE
Feature Request: Ability to exclude genres

### DISCUSSION THREAD

**Discussion by vongyver** [sct/overseerr#1652] [Feature Requests]
I am a long time user of Sonarr/Radarr and new to Overseerr and it's amazing!! 
We are not big Anime or Horror fans in the house, so it would be cool if we could filter out recommendations of those two genres.  I know that would require a new or extended UI for this and a filter, but I think it would allow users to focus on the options.  If this happens automagically over time, please disregard. 

I will offer for testing, if valuable.  I will also look at the code and see if I can figure it out myself, but I am old junior programmer at best. 

Kudos to the dev team, great work!!

**Comment by TheCatLady** [comment]
Hi @vongyver, glad you like Overseerr so far!

We do plan to add additional filtering options to the UI.  You can thumbs up and subscribe to this feature request for updates: #459 (please do not comment unless you have something constructive to add).

P.S.- Consider [joining us on Discord](https://discord.gg/PkCWJSeCk7)!

**Comment by vongyver** [comment]


**Reply by TheCatLady** [reply]
@vongyver Huh, that's odd.  I just checked the link again and it is still valid.

**Comment by vongyver** [comment]
Hopefully I catch you before you work on it. My discord app was crashed in
the background and kill it and restarting it resolved it.
I'm in now.

Thanks!

**Comment by Starminder** [comment]
Would love for this feature to appear.   I spend a lot of time looking for new shows, and often it is because there is so much anime.
